src/app/app.module.ts

```diff
@@ -58,6 +58,22 @@ import { RedirComponent } from './redir/redir.component';
 import { UpdateRoleComponent } from './update-role/update-role.component';
 import { CardComponent } from './card/card.component';
 
+import { ComplaintsHomeComponent } from './venter/complaints-home/complaints-home.component';
+import { MatTabsModule } from '@angular/material/tabs';
+import { MatCardModule } from '@angular/material/card';
+import { MatGridListModule } from '@angular/material/grid-list';
+import { MatListModule } from '@angular/material/list';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { FileComplaintComponent } from './venter/file-complaint/file-complaint.component';
+import { MatInputModule } from '@angular/material';
+import { AgmCoreModule } from '@agm/core';
+import { MatAutocompleteModule } from '@angular/material/autocomplete';
+import { MatChipsModule } from '@angular/material/chips';
+import { MatButtonModule } from '@angular/material/button';
+import { MatDialogModule } from '@angular/material';
+import { DetailedComplaintComponent } from './venter/detailed-complaint/detailed-complaint.component';
+import { MatDividerModule } from '@angular/material/divider';
```

Comment:
All these imports don't belong here. Most of them are already imported in `material-angular.module.ts`. Any extra `@angular/material` module imports should be placed there.